defaults/main.yml

```
@@ -1,5 +1,9 @@
 clamav_cron_frequency: weekly
 
+apparmor_clamd_configuration_path: /etc/apparmor.d/usr.sbin.clamd
+apparmor_freshclam_configuration_path: /etc/apparmor.d/usr.bin.freshclam
+apparmor_complain: true
```

I'd like this to be `false` by default, since the AppArmor changes you make are only necessary for some use cases:
```suggestion
apparmor_complain: false
```